rs/http_endpoints/public/tests/common/mod.rs

```diff
@@ -72,6 +72,8 @@ use tokio::{
 use tower::{util::BoxCloneService, Service, ServiceExt};
 use tower_test::mock::Handle;
 
+const ENABLE_V3_CALL_HANDLER: bool = true;
```

Comment:
nit: remove the constant from here. I think it is confusing to have two constants with the same name. You can use hardcode the value when you instantiate the testing objects.